packages/proxyIdentity/contracts/IOwned.sol

```diff
@@ -0,0 +1,5 @@
+pragma solidity 0.8.3;
```

Comment:
I think it would make sense to have a more flexible pragma here to allow the interface to be used by more consumers. I picked `0.6.0` as the lower bound but maybe we can go even lower, I think we should go as low as possible
```suggestion
pragma solidity >0.6.0 <0.9.0;
```